src/calculateRentalCost.js

```diff
@@ -29,6 +29,16 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const costs = 40;
+  const discountLow = 20;
+  const discountHigh = 50;
+  const maxDayWithoutDiscount = 3;
+  const maxDayForLowDiscount = 7;
+  const costPerDay = costs * days;
```

Comment:
name does not refer to the purpose
```suggestion
  const totalCost = costs * days;
```